doc/source/user_guide/getting_help.rst

```diff
@@ -28,6 +28,24 @@ The ``quick search`` field located in the navigation bar of the html
 documentation can be used to search for specific keywords (segmentation,
 rescaling, denoising, etc.).
 
+API Discovery
+-------------
+
+The users can make use of ``np.lookfor`` method to check the functions 
```

Comment:
"to check the functions in scikit-image" <- Maybe rephrase and/or enrich this?
I've never used this function, and the result is not clear to me at all:
```
In [8]: np.lookfor('boundaries', skimage)
Search results for 'boundaries'
-------------------------------
skimage.segmentation.find_boundaries
    Return bool array where boundaries between labeled regions are True
skimage.segmentation.mark_boundaries
    Return image with boundaries between labeled regions highlighted.
skimage.future.graph.rag_boundary
    Comouter RAG based on region boundaries
skimage.segmentation.boundaries._find_boundaries_subpixel
    Notes
skimage.transform.warp
    Warp an image according to a given coordinate transformation.
skimage.transform.swirl
    Perform a swirl transformation.
skimage.filters.gaussian
    Multi-dimensional Gaussian filter
skimage.transform.resize
    Resize image to match a certain size.
skimage.transform.rotate
    Rotate image by a certain angle around its center.
skimage.transform.rescale
    Scale image by a certain factor.
skimage.morphology.watershed
    Return a matrix labeled using the watershed segmentation algorithm
```